Resources/public/js/views/ez-usermenuitemfireeventview.js

```diff
@@ -42,6 +44,27 @@ YUI.add('ez-usermenuitemfireeventview', function (Y) {
         },
 
         /**
+         * Adds the event listener
+         *
+         * @method _addEventListener
+         * @protected
+         * @param event {Object} event facade
+         */
+        _addEventListener: function (event) {
+            this.on(this.get('eventName'), this._hideUserMenu, event.userMenu);
```

Comment:
that's a very confusing way of writing that `event.userMenu.hide` should be called. I would write something like:
```js
this.on(this.get('eventName'), Y.bind(event.userMenu.hide, event.userMenu));
```